src/cli/README.md

```diff
@@ -210,6 +212,64 @@ Set the Thread Child Timeout value.
 Done
 ```
 
+### commissioner start \<psdk\> \<provisioningUrl\>
+
+Start the Commissioner role.
+
+* pskd: Pre-Shared Key for the Joiner.
+* provisioningUrl: Provisioning URL for the Joiner (optional).
+
+This command will cause the device to send LEAD_PET and LEAD_KA messages.
+
+```bash
+> commissioner start ot-psk
+Done
+```
+
+### commissioner stop
+
+Stop the Commissioner role.
+
+This command will cuase the device to send LEAD_KA messages.
```

Comment:
Then perhaps clarify it will send a LEAD_KA(Reject), because keep alives are sent while the Commissioner is alive as well.